test/feature/steps/eval.js

```diff
@@ -15,85 +19,169 @@ const SECONDS = 7;
 const defaultTimeoutTime = SECONDS * MSECONDS;
 setDefaultTimeout(defaultTimeoutTime);
 
```

Comment:
This steps file requires the comment as well.

Reply:
Done, sir.